assignment/index.html

```diff
@@ -65,7 +65,31 @@
       var jsonToCsv = function(json) { console.log(json); };
 
 
-      var addMarkers = function(map) {};
+      var addMarkers = function(map) {
+        for(var obj in healthCenters){
+
+          var lat = healthCenters[obj].LAT
+          var lng = healthCenters[obj].LNG
+
+          if (healthCenters[obj].ZIP >= 19140 && healthCenters[obj].ZIP <= 19149){
+            if (healthCenters[obj].DENTAL_PHONE != "N/A"){
+              var mrkr = L.marker([lat,lng]).addTo(map);
+            } else{
+              var myIcon = L.icon({
+                iconUrl: 'marker-2.png',   //Icon just shows up as an empty box
```

Comment:
![image](https://user-images.githubusercontent.com/8103418/52183574-0c572380-27d7-11e9-8e67-01f42294c049.png)
![image](https://user-images.githubusercontent.com/8103418/52183575-1547f500-27d7-11e9-8ade-86334cf8d290.png)

Reply:
You needed to add the `images/` before the marker location so it knows to look in a different folder.